pkgs/development/libraries/vtk/generic.nix

```diff
@@ -94,7 +78,6 @@ in stdenv.mkDerivation {
   ] ++ optionals enableQt [
     "-DVTK_GROUP_ENABLE_Qt:STRING=YES"
   ]
-    ++ optionals stdenv.hostPlatform.isDarwin [ "-DOPENGL_INCLUDE_DIR=${OpenGL}/Library/Frameworks" ]
```

Comment:
Does the build still find the OpenGL framework successfully? (If so, then this is fine.)

Reply:
```
-- Found OpenGL: /nix/store/amdympl4rz7kj93j82cva60v8007n4nv-apple-sdk-11.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework  found components: OpenGL
```